src/posit/__init__.py

```diff
@@ -1,7 +1,5 @@
 """The Posit SDK."""
 
-from . import connect  # noqa
+from . import connect as connect
```

Comment:
What does `import connect as connect` do?

Reply:
https://docs.astral.sh/ruff/rules/unused-import/

But, I now see I can also remove the __all__ statement. Going to update that now.